### ISSUE TITLE
Default Users Scaffold files missing

### ISSUE THREAD

**Issue by simocsdk** [InfyOmLabs/stisla-templates#1]
https://www.infyom.com/open-source/laravelgenerator/docs/8.0/generator-commands#default-users-model-crud

> php artisan infyom.publish:user

is not working cause Users scaffold files for this template are missing.

```
ErrorException 

copy(PATH/vendor/infyomlabs/stisla-templates/templates/scaffold/users/create.stub): failed to open stream: No such file or directory
```

**Comment by brian123434343434** [comment]
copy(/var/www/html/stisla-generator/vendor/infyomlabs/stisla-templates/templates/scaffold/users/create.stub): Failed to open stream: No such file or directory

**Comment by ShaileshInfyom** [comment]
this issue has been fixed on PR #2
